x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/filters/filters_section.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { type Dispatch, memo, type SetStateAction, useCallback, useMemo } from 'react';
+import type { Filter, TimeRange } from '@kbn/es-query';
+import { TableId } from '@kbn/securitysolution-data-table';
+import type { FilterGroupHandler } from '@kbn/alerts-ui-shared';
+import type { DataView, DataViewSpec } from '@kbn/data-views-plugin/common';
+import { PageFilters } from './page_filters';
+import type { AssigneesIdsSelection } from '../../../../common/components/assignees/types';
+import { useDataTableFilters } from '../../../../common/hooks/use_data_table_filters';
+import { useDeepEqualSelector } from '../../../../common/hooks/use_selector';
+import { useGlobalTime } from '../../../../common/containers/use_global_time';
+import { inputsSelectors } from '../../../../common/store/inputs';
+import {
+  buildAlertAssigneesFilter,
+  buildShowBuildingBlockFilter,
+  buildThreatMatchFilter,
+} from '../../alerts_table/default_config';
+import type { Status } from '../../../../../common/api/detection_engine';
+
+export interface FiltersSectionProps {
+  /**
+   * List of assignees retrieved from the assignees button on the alert page
+   */
+  assignees: AssigneesIdsSelection[];
+  /**
+   * Data view used for the alerts page
+   */
+  dataView: DataView | DataViewSpec;
```

Comment:
You can remove `DataViewSpec` here. On the attacks page we're guaranteed to always have a `DataView` as the page does not exist if the `newDataViewPickerEnabled` feature flag is disabled.